csi.proto

```diff
@@ -1163,6 +1201,13 @@ message Snapshot {
   // `volume_content_source` in a `CreateVolumeRequest`. The default
   // value is false. This field is REQUIRED.
   bool ready_to_use = 5;
+
+  // The ID of the volume group snapshot that this snapshot is part of.
+  // This field is only valid if the plugin has implemented the
+  // CREATE_DELETE_VOLUME_GROUP_SNAPSHOT capability in the
+  // GroupControllerService.
+  // This field is OPTIONAL.
+  string group_snapshot_id = 6 [(alpha_field) = true];
```

Comment:
@xing-yang  afaict, this field has to carry the information to uniquely identify this volume group snapshot. if agreed, we have to document the same .